test/e2e/restart.go

```diff
@@ -30,6 +31,29 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func isRestartNeverMirrorPod(p *api.Pod) bool {
+	_, ok := p.Annotations[kubetypes.ConfigMirrorAnnotationKey]
```

Comment:
nit: Can we use [`IsMirrorPod`](https://github.com/kubernetes/kubernetes/blob/8fd414537b5143ab039cb910590237cabf4af783/pkg/kubelet/pod/mirror_client.go#L97)?